src/ProjectTask.php

```diff
@@ -504,6 +504,8 @@ public function prepareInputForUpdate($input)
             $input['_old_projecttasks_id'] = $this->fields['projecttasks_id'];
         }
 
+        $input = self::autoSetDate($input, $this->fields);
```

Comment:
We try to avoid static methods (here, it also avoid sending the fields as they are already available in the item).

```suggestion
        $input = self->autoSetDate($input);
```